Makefile

```
@@ -298,12 +298,12 @@ dev: push-image task-bundle ## Push the ec-cli and v-e-c Task Bundle to the kind
 TASK_TAG ?= latest
 TASK_REPO ?= quay.io/enterprise-contract/ec-task-bundle
 TASK_VERSION ?= 0.1
-TASKS ?= tasks/verify-enterprise-contract/$(TASK_VERSION)/verify-enterprise-contract.yaml
+TASKS ?= tasks/verify-enterprise-contract/$(TASK_VERSION)/verify-enterprise-contract.yaml,tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml
```

```suggestion
TASKS ?= tasks/verify-enterprise-contract/$(TASK_VERSION)/verify-enterprise-contract.yaml,tasks/verify-conforma-konflux-ta/$(TASK_VERSION)/verify-conforma-konflux-ta.yaml
```

or...

```suggestion
TASK_VERSION_TA ?= 0.1
TASKS ?= tasks/verify-enterprise-contract/$(TASK_VERSION)/verify-enterprise-contract.yaml,tasks/verify-conforma-konflux-ta/$(TASK_VERSION_TA)/verify-conforma-konflux-ta.yaml
```

---

Seems TASK_VERSION is used nowhere else, so you could also remove it and do:

```suggestion
TASKS ?= tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml,tasks/verify-conforma-konflux-ta/0.1/verify-conforma-konflux-ta.yaml
```

---

removed!